parsers/requirements.txt

```diff
@@ -1,5 +1,6 @@
 arrow==0.10.0
 beautifulsoup4==4.5.3
+demjson==2.2.4
```

Comment:
why use demjson? Feel free to answer @jarek 's question (he asked the same).
I'd be in favour of limiting dependencies if we can.

Reply:
jarek has it right, this stuff is actually javascript not valid json.  The json module cannot deal with it properly, more info here https://stackoverflow.com/questions/34812821/python-bad-json-keys-are-not-quoted